app/adapters/hathi/metadata_adapter.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+require "mini_magick"
+
+module Hathi
+  class MetadataAdapter
```

Comment:
I think this ingester is probably fine (I haven't looked too deep), but this definitely isn't a `Valkyrie::MetadataAdapter`. As such, it's probably easier to just name this abstraction something else. `Hathi::Image` maybe?

Reply:
`Hathi::ImageMetadata`?

Reply:
Also, should it be moved out of the `adapter` directory? Maybe move it to a new directory called `exporters`.